catalog/core/catalog-core-impl/filter-proxy/src/test/java/ddf/catalog/filter/proxy/adapter/test/FilterAdapterTest.java

```diff
@@ -603,7 +606,7 @@ public void testTemporalRelativeFilter() {
     doReturn("created").when(mockPropertyName).accept(any(), any());
 
     Literal mockLiteral = mock(Literal.class);
-    doReturn("RELATIVE(P2Y3M2DT1H6M)").when(mockLiteral).accept(any(), any());
+    doReturn("RELATIVE(P3Y3M2DT1H6M)").when(mockLiteral).accept(any(), any());
```

Comment:
This got modified while I was playing with the code I'll change it back if there's some other change I need to go in and make.